Makefile

```diff
@@ -1,13 +1,17 @@
 SHELL=/bin/bash
 DATETIME:=$(shell date -u +%Y%m%dT%H%M%SZ)
 
+help: # preview Makefile commands
+	@awk 'BEGIN { FS = ":.*#"; print "Usage:  make <target>\n\nTargets:" } \
+/^[-_[:alpha:]]+:.?*#/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
 ## ---- Dependency commands ---- ##
 
-install: # install dependencies
+install: # install Python dependencies
```

Comment:
Might be worth double checking, but I _think_ maybe these need to be double ampersands `#` to show-up in the `make help` command?

e.g.
```
install: ## install Python dependencies
```

Could be wrong, but I think I experienced this behavior when adding `make help` to the GeoHarvester.

Reply:
Ooh, I updated the code I copied very slightly to work with single hashes, so it should work!

Reply:
Ah nice, good call!